op-e2e/actions/upgrades/isthmus_fork_test.go

```diff
@@ -284,3 +286,85 @@ func verifyIsthmusHeaderWithdrawalsRoot(gt *testing.T, rpcCl client.RPC, header
 		require.Equal(gt, *header.WithdrawalsHash, storageHash)
 	}
 }
+
+func TestIsthmusNetworkUpgradeTransactions(gt *testing.T) {
+	t := helpers.NewDefaultTesting(gt)
+	dp := e2eutils.MakeDeployParams(t, helpers.DefaultRollupTestParams())
+	isthmusOffset := hexutil.Uint64(4)
+
+	log := testlog.Logger(t, log.LevelDebug)
+
+	zero := hexutil.Uint64(0)
+
+	// Activate all forks at genesis, and schedule Ecotone the block after
+	dp.DeployConfig.L2GenesisHoloceneTimeOffset = &zero
+	dp.DeployConfig.L2GenesisIsthmusTimeOffset = &isthmusOffset
+	dp.DeployConfig.L1PragueTimeOffset = nil
```

Comment:
Why not activate prague on L1? Wouldn't that make a more realistic test?

Reply:
also already updated!